ttnn/cpp/pybind11/device.cpp

```diff
@@ -183,13 +187,37 @@ void device_module(py::module& m_device) {
                 Returns:
                     SubDeviceManagerId: The ID of the created sub-device manager.
             )doc")
+        .def(
+            "create_sub_device_manager_with_fabric",
+            [](Device* device, const std::vector<SubDevice>& sub_devices, DeviceAddr local_l1_size) {
+                std::tuple<SubDeviceManagerId, SubDeviceId> sub_device_manager_id_and_fabric_sub_device_id;
+                device->push_work(
+                    [device, sub_devices, local_l1_size, &sub_device_manager_id_and_fabric_sub_device_id] {
+                        sub_device_manager_id_and_fabric_sub_device_id =
+                            device->create_sub_device_manager_with_fabric(sub_devices, local_l1_size);
+                    },
+                    true);
```

Comment:
`/*blocking=*/true` here

Reply:
Also `sub_device_manager_id_and_fabric_sub_device_id` could be shorter, something like `manager_and_sub_device_ids`?

Reply:
Done